test/src/unit-capi-array.cc

```diff
@@ -117,6 +118,20 @@ std::string ArrayFx::random_name(const std::string& prefix) {
   return ss.str();
 }
 
+int ArrayFx::get_fragment_timestamps(const char* path, void* data) {
```

Comment:
That could return the result vector directly, as the timestamps will always be `uint64_t`.

Reply:
Yes but this is a callback routine for tiledb_vfs_ls so I have to respect the signature.